internal/server/data/passwordresettoken.go

```diff
@@ -17,6 +18,7 @@ func CreatePasswordResetToken(db *gorm.DB, user *models.Identity) (*models.Passw
 	}
 
 	prt := &models.PasswordResetToken{
+		ID:         uid.New(),
```

Comment:
I forgot to say in the commit message, but this was to fix a test failure. On the org_collab branch we removed all the required not-null for primary keys. I did not include that part, so we need to set an primary key for this type before saving it.  It does not use `Model`, so it doesn't get set that way.